distributions/rhel-84.json

```diff
@@ -12,6 +12,9 @@
     }, {
       "baseurl": "https://cdn.redhat.com/content/dist/rhel8/8/x86_64/appstream/os",
       "rhsm": true
+    }, {                                                                                                                                                      
+      "baseurl": "https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/ansible/2.9/os",                                                                   
+      "rhsm": true                                                                                                                                            
```

Comment:
Hmm when I enable it on a rhel-8.4 vm (`subscription-manager repos --enable ansible-2-for-rhel-8-x86_64-rpms`) I get:

```Repo ID:   ansible-2-for-rhel-8-x86_64-rpms
Repo Name: Red Hat Ansible Engine 2 for RHEL 8 x86_64 (RPMs)
Repo URL:  https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/ansible/2/os
Enabled:   1
```

Is there a reason to pin it to `2.9?`

Reply:
Agreed and changed